acceptance_tests/features/edit_respondent_details.feature

```diff
@@ -43,46 +44,23 @@ Feature: As an internal user
 
   @us058-s001
   Scenario: The user is able to edit the email address of a respondent
-    Given the respondent with email "test_respondent@test.com" is enrolled
-    And the internal user has found the respondents details
+    Given the respondent with email "test_respondent2@test.com" is enrolled
+    And the internal user has found the second respondents details
     When they change the email address
     And they click save
     Then they are able to save the updated email address
+    And they are presented with confirmation that the changes have been saved
 
   @us058-s002
-  Scenario: The user is able to enter up to 254 characters for an email address
```

Comment:
How comen we're deleting these tests?

Reply:
@us058-s001, @us058-s002 and @us058-s003 were functionally all doing the same thing, I can add them back in but it's not testing any new functionality.

@us058-s007 was the pretty much identical as well and is now covering in the final `And` step added to @us058-s001

Reply:
Let's just keep them separate I think. Smaller more specific tests give better feedback and generally live by the rule of only testing one thing. This doesn't come without downsides like speed but they can be mitigated. Our test time isn't actually a concern at the moment as they run in ~ 5 minutes

After reading the stories they are poorly written and should have been refined before now.

I think `Scenario: The user is able to enter up to 254 characters for an email address` is a technical requirement so shouldn't really be an acceptance criteria.

In the scenario `Scenario: The user is able to edit the email address of a respondent`
`When they change the email address` and `Then they are able to save the updated email address` doesn't describe at all what the user is expecting to see it's just rewording the `when`

Reply:
:+1: definitely agree on them being poorly written I'll try and separate them out where appropriate then